
#defines timeout to exit the tracking (in milliseconds)
#int
 poseTimeoutToExit: 2000

#sets the visibility parameter for skeletons of the tracked users
#bool 
 drawSkeleton: false

#sets the visibility parameter for a center of tracked users mass
#bool 
 drawCenterOfMass: false

#sets the visibility parameter for names of the tracked users
#bool 
 drawUserName: false

#sets the visibility parameter for bounding boxes of the tracked users
#bool 
 drawBoundingBox: true

#sets the visibility parameter for a background
#bool 
 drawBackground: true

#sets the visibility parameter for a detected depth data (if true, if publishes the original point cloud with all points that belong to one tracked person colored in a individual color on topic "points_body_tracker")
#bool 
 drawDepth: true

# publishes the detected frames on tf and some Rviz markers for display (sets the visibility parameter for skeleton frames of the tracked users)
#bool 
 drawFrames: true
